test/lib/usd/translators/UsdExportStagesAsRefsTest/sceneWithStage.ma

```diff
@@ -2,12 +2,13 @@
 //Name: sceneWithStage.ma
 //Last modified: Wed, Jun 26, 2024 04:41:34 PM
 //Codeset: 1252
+requires maya "2019ff01";
 requires -nodeType "mayaUsdLayerManager" -nodeType "mayaUsdProxyShape" -dataType "pxrUsdStageData"
 		 "mayaUsdPlugin" "0.30.0";
 currentUnit -l centimeter -a degree -t film;
 fileInfo "application" "maya";
-fileInfo "product" "Maya 2022";
-fileInfo "version" "Preview Release 123";
+fileInfo "product" "Maya 2019";
```

Comment:
Is there a reason for this change? We don't support Maya 2019 anymore.

Reply:
There was a "requires" missing for the Maya version, above this. I took the requires line from another file we had in our unit tests and I thought it was better if the "requires" matched the fileInfo data, just in case it matters.

Reply:
I think it would be better to leave it at 2022 since the original file was. This should work:
```
requires maya "2022";
fileInfo "product" "Maya 2022";
fileInfo "version" "2022";
```